features/pixel-segmentation-comparison-plugin/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM polusai/bfio:2.1.9
+
+# environment variables defined in polusai/bfio
+ENV EXEC_DIR="/opt/executables"
+ENV POLUS_IMG_EXT=".ome.tif"
+ENV POLUS_TAB_EXT=".csv"
+
```

Comment:
Need to add `POLUS_LOG` to the environment variables.

Reply:
done